.flowconfig

```diff
@@ -6,26 +6,38 @@
 # - https://flowtype.org/docs/quick-reference.html
 # - https://flowtype.org/docs/objects.html
 # - https://flowtype.org/docs/functions.html
-.*/node_modules/.*
+.*/Gruntfile.js
 .*/dist/.*
-.*/ignored/.*
-.*/historical/.*
 .*/frontend/assets/.*
-.*/frontend/controller/backend/interface.js
+.*/frontend/controller/service-worker.js
 .*/frontend/controller/utils/primus.js
+.*/frontend/utils/blockies.js
 .*/frontend/utils/crypto.js
-.*/frontend/utils/vuexQueue.js
 .*/frontend/utils/flowTyper.js
-# .*/test/backend.js
+.*/frontend/utils/vuexQueue.js
+.*/frontend/views/containers/chatroom/.*
+.*/historical/.*
```

Comment:
Can you also add:

```
.*/ignored/.*
```

Reply:
It was there initially, but I noticed that no top-level `ignored` folder existed, so I thought it would be better to remove it; but I can add it back of course.

Reply:
Thanks! Yes it does exist on my machine, and anyone else's machine who wants such a folder. 😄